tests/foreman/longrun/test_n_1_upgrade.py

```diff
@@ -0,0 +1,262 @@
+"""Test class for N-1 upgrade feature
+
+:Requirement: Satellite one version ahead from Capsule
+
+:CaseAutomation: ManualOnly
+
+:CaseLevel: Integration
+
+:CaseComponent: Capsule
+
+:TestType: Functional
+
+:CaseImportance: Critical
+
+:Upstream: No
+"""
+import pytest
+
+from robottelo.decorators import tier3
+
+
+@pytest.mark.stubbed
+@tier3
+def test_n_1_setup():
+    """
+    Prepare the environment to test the N-1 Capsule sync scenarios.
+
+    :id: 62c86858-4803-417a-80c7-0070df228355
+
+    :steps:
+        1. Login Satellite with admin rights.
+        2. Add the Red Hat and Custom Repository.
+        3. Create the lifecycle environment.
+        4. Create the Content view and select all the repository.
+        5. Publish the content view and promote the content view with created lifecycle
+            environment.
+        6. Create the activation key and add the content view.
+        7. Add the subscription, if it is missing in the activation key.
+
+    :expectedresults: Initial setup should complete successfully
+
+    :CaseImportance: Critical
+
+    :CaseLevel: System
+
+    :CaseAutomation: ManualOnly
```

Comment:
It's already set on module level, no need to repeat it here for each individual test.